src/qililab/instruments/qblox/qblox_module.py

```diff
@@ -96,6 +96,7 @@ def initial_setup(self):
             sequencer_id = sequencer.identifier
             # Set `sync_en` flag to False (this value will be set to True if the sequencer is used in the execution)
             self.device.sequencers[sequencer_id].sync_en(False)
+            self.device.sequencers[sequencer_id].marker_ovr_en(False)
```

Comment:
For me to understand, waht does it mean to set the sequencer marker override enable to False? Are all the markers deactivated, all enabled?

Reply:
By default all markers are OFF, so the value is "0000". The `marker_ovr_en(False)` makes sure that markers' value are not overrided by QCoDes by default, so they can be set in Q1ASM.